<?xml version="1.0" ?>

<launch>
  <!-- SONAR -->
  <node name="mb1200_beagle_driver" type="mb1200_beagle_driver.py" pkg="falkor_drivers" >
    <param name="tf_prefix" value="robot" />
  </node>

  <!-- I2C -->
  <node output="screen" name="i2c_driver" type="i2c_driver.py" pkg="falkor_drivers" >
    <param name="tf_prefix" value="robot" />
    <rosparam param="mag_calib">
      center: { x: 0.13213062, y: -0.10308636" }
      axes: { x: 1, y: 1.143700 }
      angle: 0.14061404
    </rosparam>
  </node>

  <!-- PWM -->
  <node name="pwm_io" type="pwm_io_node" pkg="falkor_drivers" />


  <!-- PWM Controller -->
  <node name="falkor_control_pwm" type="falkor_control_pwm.py" pkg="falkor_quadrotor_control" >
    <param name="tf_prefix" value="robot" />
  </node>

  <!-- NMEA Gps (not rtklib) -->
  <node name="nmea_gps_driver" type="nmea_gps_driver.py" pkg="nmea_gps_driver" >
    <param name="port" value="/dev/ttyO2" />
    <param name="baud" value="115200" />
    <param name="tf_prefix" value="robot" />
  </node>
</launch>
